cli/boards.go

```diff
@@ -77,6 +77,39 @@ func UploadBoardDefsAction(ctx *cli.Context) error {
 	return nil
 }
 
+// GetBoardDefsAction is the corresponding action for "board get".
+func GetBoardDefsAction(c *cli.Context) error {
+	orgArg := c.String(organizationFlag)
+	nameArg := c.String(boardFlagName)
+	versionArg := c.String(boardFlagVersion)
+
+	if versionArg == "" {
+		versionArg = "latest"
+	} else if !supportedVersionRegex.MatchString(versionArg) {
+		return fmt.Errorf("invalid version %s. Must use semver 2.0.0 specification for versions", versionArg)
+	}
+
+	client, err := newAppClient(c)
+	if err != nil {
+		return err
+	}
+
+	// get the org from the name or id.
+	org, err := client.getOrg(orgArg)
+	if err != nil {
+		return err
+	}
+
+	url, err := client.getBoardDefsFile(nameArg, versionArg, org.Id)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(c.App.Writer, "Download the %s board definition file here: %s\n", nameArg, url)
```

Comment:
Heads up that this produces the following msg:
`
Download the myboard board definition file here: https://storage.googleapis.com/viam-packages-1a38b76f-cac0-451c-8f72-670994fcaf5b/1a38b76f-cac0-451c-8f72-670994fcaf5b/packages/myboard/gKAVsTgxLTPZ76u8qCKdMRas1ZMRSmrhyu63J70xW3k7ydqR8po2NpN1F4psne80.tar.gz?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=app-service%40web-app-304613.iam.gserviceaccount.com%2F20230828%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20230828T192938Z&X-Goog-Expires=899&X-Goog-Signature=07e602be2bd2631efc21673f5a927897c2d2c65122e81b6bd542884d87daa1081c16a5d1ca44958fd20d10c07131308463c528f7ec3644dd71f9c153509c1bac6919e94a6561d86641760df7529a838149927ee562a41c81b0ee464ab6383031c03f287c3fdaa7b60a7376fbf8ce2f0de2228b0fa2e081a6d384a4b197d00034bb34187924167b9bcaa8e6970c9274737ee066454f951fb4b49673bb2082148f9eba42120dfa82ceb3fe078654499db880d772ad292eedfeb9d422a286a51a5c302126ac90afa15a65494794930d7b1676ccbef2e85f8083bbb7de3ccced78acfa9fcbe9257421cb74440c85fe3b9e1e3b60c24f0c264dec44950136b09f3074&X-Goog-SignedHeaders=host
`

(and running `wget <thaturl>` produces a file called:
```
gKAVsTgxLTPZ76u8qCKdMRas1ZMRSmrhyu63J70xW3k7ydqR8po2NpN1F4psne80.tar.gz?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=app-service@web-app-304613.iam.gserviceaccount.com%2F20230828%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20230
```
)


Given the 
```
DESCRIPTION:
   download a json board definitions file for generic linux boards.
```
 I would have expected this cli command to download and extract the tar file...? 

If it has already been decided that we are not doing that for reasons I am unaware of, please feel free to resolve. Don't want to intrude.

Reply:
I like Zack's suggestion: when you uploaded a file, it was `board_defs.json`, and after you download it, I think you should get `board_defs.json` back again. I think we should download the tarball, unzip it, and then maybe delete the .tar.gz file we downloaded.

...though perhaps this is just part 1 of the functionality, that in this PR we just print out what ought to happen in part 2, and we should expect an upcoming PR that takes that URL, and downloads and extracts the tarball?

Reply:
we hadn't discussed it, just added functionality to download the file from the gcs link for the user.